java/java.hints/test/unit/src/org/netbeans/modules/java/hints/errors/AddFinalModifierTest.java

```diff
@@ -105,4 +107,12 @@ protected String toDebugString(CompilationInfo info, Fix f) {
     protected Set<String> getSupportedErrorKeys() {
 	return AddOrRemoveFinalModifier.createAddFinalModifier().getCodes();
     }
+    public static Test suite() {
+        //javac no longer supports any source level that would produce this error
+        return new TestCase("noop") {
+            @Override
+            public void runBare() throws Throwable {
+            }
+        };
+    }
```

Comment:
i suppose that is because variables are now implicitly final if they are only read so that they can be used in lambdas etc. Is it still useful to keep this test around?